packages/manager/apps/catalog-revamp/src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useRef, useEffect } from 'react';
+import {
+  OsdsInput,
+  OsdsButton,
+  OsdsIcon,
+} from '@ovhcloud/ods-stencil/components/react/';
+import {
+  OdsIconName,
+  OdsIconSize,
+  OdsButtonVariant,
+  OdsButtonType,
+  OdsButtonSize,
+  OdsInputType,
+  OdsInputValueChangeEventDetail,
+} from '@ovhcloud/ods-core';
+import { OdsThemeColorIntent } from '@ovhcloud/ods-theming';
+import { useTranslation } from 'react-i18next';
+import Filters from '@/components/Filters/Filters';
+import { Product } from '@/hooks/useCatalog';
+
+import './SearchBar.scss';
+
+const SearchBar: React.FC = () => {
+  const [searchValue, setSearchValue] = useState('');
+
+  const { t } = useTranslation('catalog-revamp/search');
+
+  const handleSearchChange = (
+    event: CustomEvent<OdsInputValueChangeEventDetail>,
+  ) => setSearchValue(event.detail.value);
+
+  return (
+    <span className="searchbar-container">
```

Comment:
**nitpick:** Since search bar is specific to catalog page, prefix it with catalog or have a `catalog` class as parent and scope all the classes inside parent class.
```suggestion
    <span className="catalog-searchbar-container">
```

Reply:
Done, I put catalog-searchbar-container catalog-searchbar catalog-search-button so we can be sure they are unique